kernel/src/scan/log_replay.rs

```diff
@@ -119,39 +157,42 @@ impl RowVisitor for AddRemoveVisitor {
 
 // NB: If you update this schema, ensure you update the comment describing it in the doc comment
 // for `scan_row_schema` in scan/mod.rs! You'll also need to update ScanFileVisitor as the
-// indexes will be off
+// indexes will be off, and `get_add_transform_expr` below to match it.
```

Comment:
```suggestion
// indexes will be off, and [`get_add_transform_expr`] below to match it.
```
Linking makes it easy to jump there and back with an lsp.